workers/unity/Packages/com.improbable.gdk.ui/DeploymentWindow.cs

```diff
@@ -0,0 +1,186 @@
+                            Debug.Log(path);
+                            Process.Start(path);
+                        }
+                    }
+
+                    if (GUILayout.Button("Runtime log"))
+                    {
+                        if (Application.platform == RuntimePlatform.WindowsEditor)
+                        {
+                            var path = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), ".improbable", "spatiald", "log", "local", "runtime.log");
+                            Debug.Log(path);
+                            Process.Start(path);
+                        }
+                    }
+                }
+
+                if (deployment == null)
+                {
+                    if (GUILayout.Button("Local launch"))
+                    {
+                        DoLocalLaunch();
+                    }
+                }
+                else if(deployment.Status == Deployment.Types.Status.Starting)
+                {
+                    if (GUILayout.Button("Cancel"))
+                    {
+                        StopDeployment(deployment);
+                    }
+                }
+                else if (deployment.Status == Deployment.Types.Status.Running)
+                {
+                    if (GUILayout.Button("Stop"))
+                    {
+                        StopDeployment(deployment);
+                    }
+                }
+
+                if (deployment != null)
+                {
+                    GUILayout.Label($"Name: {deployment.Name}");
+                    GUILayout.Label($"Id: {deployment.Id}");
+                    GUILayout.Label($"Status: {deployment.Status}");
+                }                
+            }
+        }
+
+        private async void StopDeployment(Deployment dpl)
+        {
+            var result =
+                localDeploymentClient.StopDeploymentAsync(new StopDeploymentRequest
+                {
+                    Id = dpl.Id,
+                    ProjectName = projectName
+                });
+
+            GetDeployments();
+
+            await result;
+
+            GetDeployments();
+        }
+
+        private async void DoLocalLaunch()
+        {
+            var configJson = File.ReadAllText(Path.Combine(LocalLaunch.SpatialProjectRootDir, "default_launch.json"), Encoding.UTF8);
+            localDeploymentOperation = localDeploymentClient.CreateDeployment(new CreateDeploymentRequest(
+                new CreateDeploymentRequest
+                {
+                    Deployment = new Deployment
+                    {
+                        Name = "local",
+                        ProjectName = projectName,
+                        LaunchConfig = new LaunchConfig
+                        {
+                            ConfigJson = configJson
+                        }
+                    }
+                }));
```

Comment:
Might be worth feeding back to Operate that these APIs are rather verbose.. 

For example this could be - 

`localDeploymentClient.CreateLocalDeployment(projectName, new LaunchConfig(...))`

Reply:
Unless there are tons of optional arguments that I can't see here 😬

Reply:
There are some more arguments, yeah.
They just don't provide constructor overloads, which is definitely overly verbose...